Examples/UIExplorer/NavigationExperimental/NavigationAnimatedExample.js

```diff
@@ -74,11 +69,13 @@ class NavigationAnimatedExample extends React.Component {
       <NavigationAnimatedView
         navigationState={navigationState}
         style={styles.animatedView}
-        renderOverlay={(props) => (
+        renderOverlay={(scenes, index, position, layout) => (
           <NavigationHeader
-            navigationState={props.navigationParentState}
-            position={props.position}
-            getTitle={state => state.key}
+            scenes={scenes}
+            index={index}
+            position={position}
+            layout={layout}
+            renderTitleComponent={pageState => <NavigationHeaderTitle>{pageState.key}</NavigationHeaderTitle>}
```

Comment:
Same comment here. 
There is no reason that the title only cares about the `pageState`. 
Such API is very specific thus limits the design of what the Title component can do.